queries.sql

```diff
@@ -90,3 +83,72 @@ SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id WHERE o.full_nam
 
 -- ¿Quién tiene más animales?
 SELECT o.full_name, COUNT(a.id) AS animal_count FROM owners o JOIN animals a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY animal_count DESC LIMIT 1;
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'William Tatcher' LIMIT 1)
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT a.id)
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez' LIMIT 1);
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT v.name AS vet_name, s.name AS species_name
+FROM vets v
+LEFT JOIN specializations sp ON v.id = sp.vet_id
+LEFT JOIN species s ON sp.species_id = s.id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez' LIMIT 1)
+AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+
+-- What animal has the most visits to vets?
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+GROUP BY a.name
+ORDER BY COUNT(v.visit_date) DESC
+LIMIT 1;
+
+-- Who was Maisy Smith's first visit?
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'Maisy Smith' LIMIT 1)
+ORDER BY v.visit_date
+LIMIT 1;
+
+-- Details for most recent visit: animal information, vet information, and date of visit.
+SELECT a.name AS animal_name, v.name AS vet_name, vis.visit_date
+FROM animals a
+JOIN visits vis ON a.id = vis.animal_id
+JOIN vets v ON vis.vet_id = v.id
+ORDER BY vis.visit_date DESC
+LIMIT 1;
+
+-- How many visits were with a vet that did not specialize in that animal's species?
+SELECT COUNT(*)
+FROM visits v
+LEFT JOIN specializations sp ON v.vet_id = sp.vet_id
+JOIN animals a ON v.animal_id = a.id
+WHERE sp.species_id IS NULL OR sp.species_id != a.species_id;
+
+-- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+SELECT s.name AS species_name
+FROM visits v
+JOIN animals a ON v.animal_id = a.id
+JOIN species s ON a.species_id = s.id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'Maisy Smith' LIMIT 1)
+GROUP BY s.name
+ORDER BY COUNT(*) DESC
+LIMIT 1;
+
```

Comment:
- [ ] Great job so far. However, Kindly make sure to add screenshots of each of your queries results **only**. I can see you added screenshots currently but they are not specific to the queries you have written. Kindly consider fixing this issue as it would help make your pull request more descriptive and easy to follow.

Reply:
thanks @Strangeal i will fix it, I realized I uploaded the wrong screenshots